packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -296,6 +292,7 @@ const selectInputProps = computed(() => {
         <OnyxMiniSearch
           v-if="props.withSearch"
           ref="miniSearch"
+          v-model="searchTerm"
```

Comment:
Did you remove the onAutocomplete intentionally?

Reply:
Yes, but I found a better solution. Please recheck :)